get/get.go

```diff
@@ -110,7 +168,70 @@ func (cmd *Cmd) list(ctx context.Context, client *api.Client, list runtimeclient
 		}
 	}
 
-	return nil
+	if cmd.AllProjects || !cmd.searchForName {
+		return nil
+	}
+
+	// we searched for a named object in the current project only, but did
+	// not find it there. Lets see if it was found in any other project.
+	if items.Len() == 0 {
+		// we did not find the named object in any other project, so
+		// we can return already
+		return nil
+	}
+	// we found the named object at least in one different project,
+	// so we return a hint to the user to search in these projects
+	var identifiedProjects []string
+	for i := 0; i < items.Len(); i++ {
+		// the "Items" field of a list type most often contains
+		// a slice of types, but not a slice of
+		// pointers to the type (e.g. "[]corev1.Pod" and not
+		// "[]*corev1.Pod"). We still want to check which
+		// variant (pointer or no pointer) implements the
+		// runtimeclient.Object interface.
```

Comment:
is this really needed? At least angryjet (crossplane gen) will already error out if items is a slice of pointers. I did not test any further but controller-runtime might also not like it.

Reply:
Yeah, I actually did not find any reference for this, where it says "`Items` always has to be a slice of non-pointer types". That is why we try both ways here. We can also drop the 'pointer' case and ignore an item if it is a pointer type.

Reply:
I am only checking for a non-pointer type now and skip the object if it is a pointer type.